rootfs/etc/nginx/lua/balancer/ewma.lua

```diff
@@ -108,15 +159,14 @@ function _M.sync(self, backend)
   end
 
   self.peers = backend.endpoints
-  self.ewma = {}
-  self.ewma_last_touched_at = {}
+
+  ngx.shared.balancer_ewma:flush_all()
+  ngx.shared.balancer_ewma_last_touched_at:flush_all()
```

Comment:
In a subsequent PR I'll change this to not flush all data and ensure slow start for new upstreams. I'll also set a specific `ngx.var` to the value of EWMA corresponding to currently picked upstream/endpoint.